database/queries.js

```diff
@@ -0,0 +1,45 @@
+module.exports = {
+
+  getInfo: `select user_id, user_email,
+  (
+    select array_to_json(array_agg(row_to_json(d)))
+    from (
+      select calendars.calendar_id, calendars.user_id, calendars.calendar_owner,
+      coalesce(
+          (
+          select array_to_json(array_agg(row_to_json(y)))
+          from(
+            SELECT
+            categories.category_id,
+            categories.category,
+            coalesce(
+              (
+              select array_to_json(array_agg(row_to_json(z)))
+              from(
+                SELECT
+                todoitems.id,
+                todoitems.title,
+                todoitems.description,
+                todoitems.duration,
+                todoitems.start,
+                todoitems.end_date,
```

Comment:
Very nice query.

Reply:
Hey Rob, what ip address are you talking about? I just take a look at your pull request, I saw you change the property name of the table, but i think the ETL process needs the property name matched with the \copy command

Reply:
The IP of the remote host, which we're all supposed to be connecting with. It was in the last version of the controller I had in a PR: 54.209.199.189. There will be some conflicts between yours and mine. If your process is tested, I'm happy not to mess with it--that's what I was saying elsewhere.

Reply:
This is why I shared the password last night?

Reply:
oh, I didn't realize it was 54.209.199.189 before, I need to test it locally since it is still developing. I can change it back, no big deal

Reply:
That's my only issue. You probably didn't need to close it. Think would have given you the option to choose when you got to merge conflicts.